ground/gcs/src/plugins/coreplugin/boardmanager.cpp

```diff
@@ -118,6 +118,16 @@ void BoardManager::aboutToRemoveObject(QObject *obj)
         m_boardTypesList.removeAt(m_boardTypesList.indexOf(board));
 }
 
+IBoardType *BoardManager::getBoard(int type)
```

Comment:
Nice factoring.  I think there's other places that do a similar thing we can fix to use this ultimately.